queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered IS true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
```

Comment:
- [ ] Kindly fix this query as it's not correct as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=List%20name%20and%20escape%20attempts%20of%20animals%20that%20weigh%20more%20than%2010.5kg). The query should select animals with weight more than 10.5kg.

Reply:
Oops my bad, I copied the code halfway.

Thanks for pointing out my shortcoming.